apps/definition_events/lib/events.ex

```diff
@@ -12,6 +12,7 @@ defmodule Events do
     {"load:persist:end", Load.Persist},
     {"schedule:start", Schedule},
     {"schedule:end", Schedule},
+    {"definition:delete", Delete},
```

Comment:
definition is just a naming convention for the apps themselves that define data structures; this should just be "delete" or maybe "dataset:delete".

Reply:
I'm cool with "dataset:delete"